src/Pos/Delegation/Logic/VAR.hs

```diff
@@ -341,7 +361,11 @@ dlgVerifyBlocks blocks = do
         HashSet StakeholderId ->
         Block ssc ->
         ExceptT Text (MapCede (StateT DlgVerState m)) DlgUndo
-    verifyBlock _ (Left _) = [] <$ (dvCurEpoch .= HS.empty)
+    verifyBlock _ (Left genesisBlk) = do
+        dvCurEpoch .= HS.empty
+        let blkEpoch = genesisBlk ^. epochIndexL
+        noLongerRichmen <- getNoLongerRichmen blkEpoch
+        catMaybes <$> mapM getPsk noLongerRichmen
```

Comment:
It seems easy to do, so I will do it even if it's not necessary (I don't want to rely on it when it's avoidable).